README.md

```diff
@@ -614,6 +614,7 @@ The differences are
 
 1. `scopes` is only present for OAuth Apps
 2. `refreshToken`, `expiresAt`, `refreshTokenExpiresAt` are only present for GitHub Apps, and only if token expiration is enabled
+3. `onTokenCreated` is only present for OAuth Apps and GitHub Apps
```

Comment:
you can remove that line, my understanding is the option behaves the same for both OAuth and GitHub Apps?

Reply:
```suggestion
```